src/index.js

```diff
@@ -1,95 +1,96 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
-
-const isProduction = process.env.NODE_ENV === "production";
+import path from 'path';
+import express from 'express';
+import bodyParser from 'body-parser';
+import session from 'express-session';
+import cors from 'cors';
+import errorhandler from 'errorhandler';
+import morgan from 'morgan';
+import methodOverride from 'method-override';
+import debug from 'debug';
+import apiRoutes from './routes';
+import { serve, setup } from 'swagger-ui-express';
+import swaggerDoc from '../swaggerDoc';
+import './models/User';
+
+const isProduction = process.env.NODE_ENV === 'production';
 
 // Create global app object
 const app = express();
+const consoleLog = debug('dev');
 
 app.use(cors());
 
 // Normal express config defaults
-app.use(require("morgan")("dev"));
+app.use(morgan('dev'));
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
-app.use(require("method-override")());
-app.use(express.static(__dirname + "/public"));
+app.use(methodOverride());
+app.use(express.static(path.join(__dirname, '/public')));
 
 app.use(
-    session({
-        secret: "authorshaven",
-        cookie: { maxAge: 60000 },
-        resave: false,
-        saveUninitialized: false
-    })
+  session({
+    secret: 'authorshaven',
+    cookie: { maxAge: 60000 },
+    resave: false,
+    saveUninitialized: false
+  })
 );
 
 if (!isProduction) {
-    app.use(errorhandler());
+  app.use(errorhandler());
 }
 
-if (isProduction) {
-    mongoose.connect(process.env.MONGODB_URI);
-} else {
-    mongoose.connect("mongodb://localhost/conduit");
-    mongoose.set("debug", true);
-}
-
-require("./models/User");
+// if (isProduction) {
+//   mongoose.connect(process.env.MONGODB_URI);
+// } else {
+//   mongoose.connect('mongodb://localhost/conduit');
+//   mongoose.set('debug', true);
+// }
```

Comment:
Why is this commented out

Reply:
It throws an error since mongoose is currently not in use so I commented it out cause i don't want to delete those lines from the code base. But if I don't comment it out, it stops the server from starting.

Reply:
Please go ahead and remove it